src/main/java/dukeRobot/Duke.java

```diff
@@ -0,0 +1,104 @@
+import Duke.*;
+import Tasks.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+public class Duke {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        String name = "Duck";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" + "Hello! I'm " + name
+                + "\n" + "What can I do for you?\n" + "\n"
+                + "____________________________________________________________\n");
+        Scanner scanner = new Scanner(System.in);
+        String userInput = scanner.nextLine();
+        userInput = userInput.toLowerCase();
+        Task[] tasks = new Task[100];
+        while (!userInput.contains("bye")) {
+
+            System.out.println("____________________________________________________________\n");
+            int indexOfMark;
+            if (userInput.contains("list")) {
```

Comment:
May want to consider restructuring this block of code for easier understanding and readability. Especially with nested if statements.